src/classes/DRS_ContactMerge_CTRL.cls

```diff
@@ -0,0 +1,1036 @@
+            }
+            fieldRows.add(temp);
+        }
+
+        // Add a dummy row to add a 'Standard Fields' header before the list of standard fields
+        fieldRows.add(new FieldRow('Standard Fields', 'Standard Fields', false, 'separator'));
+
+        // Add all standard fields to the list of FieldRow in sorted manner
+        for (String fieldName: standardFieldList) {
+            addFieldComparisonRows(fieldName, standardFieldMap.get(fieldName), contacts);
+        }
+
+        // Add a dummy row to add a 'Custom Fields' header before the list of custom fields
+        fieldRows.add(new FieldRow('Custom Fields', 'Custom Fields', false, 'separator'));
+
+        // Add all custom fields to the list of FieldRow in sorted manner
+        for (String fieldName: customFieldList) {
+            addFieldComparisonRows(fieldName, customFieldMap.get(fieldName), contacts);
+        }
+
+        // Add a dummy row to add a 'System Fields' header before the list of system fields
+        fieldRows.add(new FieldRow('System Fields', 'System Fields', false, 'separator'));
+
+        // Add created by and last modified by system fields to the list of FieldRow
+        FieldRow createdByRow = new FieldRow('CreatedById', 'Created By', false, null),
+            lastModifiedByRow = new FieldRow('LastModifiedById', 'Last Modified By', false, null);
+        for (Contact c: contacts) {
+            SObject createdBy = c.getSObject('createdby');
+            SObject modifiedBy = c.getSObject('lastModifiedBy');
+            createdByRow.values.add(new Cell(c.id, (createdBy != null ? createdBy.get('name') : '') + ' ' + c.get('createdDate')));
+            lastModifiedByRow.values.add(new Cell(c.id, (modifiedBy != null ? modifiedBy.get('name') : '') + ' ' + c.get('LastModifiedDate')));
+        }
+        fieldRows.add(createdByRow);
+        fieldRows.add(lastModifiedByRow);
+
+        // If everything looks good go to "choose winning contact" step
+        this.step = 3;
+
+    }
+
+    /*******************************************************************************************************
+    * @description Selects all the values of a certain record. Used by the "Select All" command link on the page.
+    * @return void
+    */
+    public void selectDefaultRecord() {
+        String recordId = Apexpages.currentPage().getParameters().get('recordId');
+        if (recordId != null && selectedRecords.keySet().contains(recordId)) {
+            for (FieldRow row : fieldRows) {
+                if (row.showRadio) {
+                    row.selectedValue = recordId;
+                }
+            }
+        }
+    }
+
+    /*******************************************************************************************************
+    * @description Merges the winner and losers.
+    * @return PageReference The page that it redirects to. Same page user is in.
+    */
+    public PageReference mergeContacts() {
```

Comment:
This method looks identical to CON_ContactMerge_CTRL.mergeContacts(). We can't just copy the code to a new class, the original method needs to be shared here, and moved to a utility or service class if need be.